pkg/k8s/apis/cilium.io/const.go

```diff
@@ -47,6 +47,9 @@ const (
 	// PodNamespaceMetaLabels is the label used to store the labels of the
 	// kubernetes namespace's labels.
 	PodNamespaceMetaLabels = "io.cilium.k8s.namespace.labels"
+	// PodNamespaceMetaNameLabel is the label that Kubernetes automatically adds
+	// to namespaces.
+	PodNamespaceMetaNameLabel = "io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name"
```

Comment:
this could be something like:
```golang
	// PodNamespaceMetaNameLabel is the label that Kubernetes automatically adds
	// to namespaces.
	PodNamespaceMetaNameLabel = PodNamespaceMetaLabels + . + LabelMetadataName

	// LabelMetadataName is the label name which, in-tree, is used to
	// automatically label namespaces, so they can be selected easily by tools
	// which require definitive labels
	LabelMetadataName = "kubernetes.io/metadata.name"
```

Reply:
Yeas, makes sense.